web/app/controllers/bloggs_controller.rb

```diff
@@ -0,0 +1,54 @@
+class BloggsController < ApplicationController
+  http_basic_authenticate_with name: "dhh", password: "secret", except: [:index, :show]
+
+  def index
+    @articles = Blogg.all
+  end
+
+  def show
+    @article = Blogg.find(params[:id])
+  end
+
+  def new
+    @article = Blogg.new
+  end
+
+  def edit
+    puts 'EDITING'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.